js/app.js

```diff
@@ -1,29 +1,181 @@
-"use strict";
+const apiKey = "5b81342d";
+const searchForm = document.querySelector("#search-form");
+const resultsContainer = document.querySelector("#results");
+const detailsContainer = document.querySelector("#details");
+const paginationContainer = document.querySelector("#pagination");
+const modalOverlay = document.querySelector("#modal-overlay");
 
-const textDisplay = document.getElementById("text-display");
-const textEditor = document.getElementById("text-editor");
+/* waiting for user actions with the form */
+searchForm.addEventListener("submit", async (event) => {
+  event.preventDefault();
 
-window.addEventListener("keydown", actionOnEvent);
+  paginationContainer.style.display =
+    "none"; /* prevent pagination with no search result */
```

Comment:
It's generally considered better to hide/show with adding/removing classes than with direct inline style changes.

Reply:
My little rebellion )